src/repository/screens/repository.screen.js

```diff
@@ -145,7 +166,7 @@ class Repository extends Component {
     const openIssues = pureIssues.filter(issue => issue.state === 'open');
     const closedIssues = pureIssues.filter(issue => issue.state === 'closed');
 
-    const repositoryActions = [starred ? '★ Unstar' : '★ Star'];
+    const repositoryActions = ['Share', starred ? 'Unstar' : 'Star'];
```

Comment:
I think it might be more natural for "Share" to be at the bottom of the list, that's usually where I'm used to seeing it.